reporting.md

```diff
@@ -61,8 +61,52 @@ The user account that runs the Power BI installer will be the administrator of t
 ## Backfill Sync Integrations
 The backfill job uploads historical data to Power BI and keeps the reports up to date.
 
-## Simeon Sync Power BI dataset
-Each row in the data source represents a single property of a configuration.
+## Baseline and Compliance report
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/master/assets/images/power_bi.png" width: 700; height: auto;/>
+
+*This screenshot shows the baseline compliance report for a demo environment.*
+
+The Baseline Compliance Report displays how each of your tenants compare to their baseline. You can see the comparison **by tenant**, **type of configuration**, and **configuration**. You can also drill down to see how properties in a **specific configuration** compare to the baseline.
```

Comment:
Should probably standardize capital letters. We are doing all lower case in the line above for "baseline compliance report"